api/index.js

```diff
@@ -131,7 +134,8 @@ function _getMovieResume(movieId) {
             title: movie.title,
             poster: movie.poster,
             movieType: movie.movieType,
-            productionYear: movie.productionYear
```

Comment:
@pierr i don't understand how do you want me to rewrite 
‘‘‘javascript
if (movie) {
        const movieResume = {
            code: movie.code,
            title: movie.title,
            poster: movie.poster,
            movieType: movie.movieType,
            productionYear: movie.productionYear,
            linked: true
        };
        return movieResume;
    }
’’’